Formula/svt-av1.rb

```diff
@@ -0,0 +1,32 @@
+class SvtAv1 < Formula
+  desc "AV1 encoder"
+  homepage "https://gitlab.com/AOMediaCodec/SVT-AV1"
+  url "https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v1.1.0/SVT-AV1-v1.1.0.tar.gz"
+  sha256 "1c211b944ac83ef013fe91aee96c01289da4e3762c1e2f265449f3a964f8e4bc"
+  license "BSD-3-Clause"
+  head "https://gitlab.com/AOMediaCodec/SVT-AV1.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "yasm" => :build
+
+  resource("testvideo") do
+    url "https://github.com/grusell/svt-av1-homebrew-testdata/raw/main/video_64x64_yuv420p_25frames.yuv"
+    sha256 "0c5cc90b079d0d9c1ded1376357d23a9782a704a83e01731f50ccd162e246492"
+  end
+
+  def install
+    args = std_cmake_args + %W[
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+    ]
+    system "cmake", "-S", ".", "-B", "build", *args
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", "-DCMAKE_INSTALL_RPATH=#{rpath}", *std_cmake_args
```
We don't need a separate array for this.

Reply:
Thanks, PR updated according to suggestion.